update-keys.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+wget -qO- https://github.com/nodejs/node/raw/master/README.md | awk '/^gpg --keyserver pool.sks-keyservers.net --recv-keys/ {print $NF}' > keys/node.keys
```

Comment:
Think the other scripts are using curl instead of wget, but I don't know if it really needs to be consistent here

Reply:
Good catch! Should maintain the consistency ;)